nemo_curator/classifiers/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+os.environ["RAPIDS_NO_INITIALIZE"] = "1"
+from .aegis import AegisClassifier
+from .domain import DomainClassifier
+from .quality import QualityClassifier
+
+__all__ = ["DomainClassifier", "QualityClassifier", "AegisClassifier"]
```

Comment:
Need to update second cell of https://github.com/NVIDIA/NeMo-Curator/blob/main/tutorials/distributed_data_classification/distributed_data_classification.ipynb to have:
```
from nemo_curator.classifiers import DomainClassifier, QualityClassifier
```